Continuous integration.md

```diff
@@ -2,114 +2,26 @@ Continuous Integration (CI) is a development practice that requires developers t
 
 By integrating regularly, you can detect errors quickly and locate them more easily.
 
-## CircleCI
+## Bitrise
 
-CircleCI is a container-based CI service. It has multiple features:
+For all active projects we use Bitrise (previously we used CircleCI). Unlike Circle, which is suitable for any type of project, Bitrise is CI/CD just for mobile apps and it offers [some advantages](https://infinum.com/the-capsized-eight/bitrise-vs-circleci-for-android-in-a-head-to-head-battle) for mobile development CI.
 
-* support for all kinds of programming languages
-* free of charge for the first container
-* support for static code checkers (Lint, Findbugs, PMD, Checkstyle)
-* no additional configuration is needed
-* everything is maintained and updated by their support team
+Bitrise and Circle are a cloud-based solution, your code gets cloned to a remote server, which could pose a security problem for your client.
 
-However, CircleCI currently supports only GitHub repositories, and it uses the default Android emulator, so your execution times are a little bit longer than expected.  You'll also need admin privileges to the GitHub's repository which you want to connect CircleCI to.
+## Configuration
 
-Also, as CircleCI is a cloud-based solution, your code gets cloned to a remote server, which could pose a security problem for your client.
+We usually have two main workflows:
 
-## Configuration
+- One that's run for every push and contains all the checks and linters. Status of this workflow is reported back to Github, so Pull Request can't be merged if these checks don't pass. That workflow might look something like this:
+
+![aa](https://i.imgur.com/mv0fCWS.png)
+
+- One that's run to distribute the app to internal store or Google Play
 
-CircleCI uses a simple configuration script, which is placed in the `.circleci/config.yml` file:
-
-```yml
-# Build configuration file for Circle CI
```

Comment:
I don't think we need to post our Bitrise.yml. It's long file and there's nothing too much different from default .yml generated by Bitrise.

Reply:
Yeah, makes sense!